### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by ljiatu**
Most of the typos are in comments, but there are two in the actual code.

### CODE REVIEW

File: contracts/BasePoolConfig.sol [hunk 1]
```diff
@@ -79,22 +79,22 @@ contract BasePoolConfig is Ownable, Initializable {
...
-    // Evaluation Agents (EA) are the risk underwriting agents that associated with the pool.
+    // Evaluation Agents (EA) are the risk underwriting agents that are associated with the pool.
     address public evaluationAgent;
 
     uint256 public evaluationAgentId;
 
     PoolConfig internal _poolConfig;
 
-    AccruedIncome internal _accuredIncome;
+    AccruedIncome internal _accruedIncome;
```
Comment (ljiatu) [inline contracts/BasePoolConfig.sol:89]: I'm seeing these names in HTML files too. Should I update those directly or through some sort of file generation tool?

Comment (PlayJok3r) [reply contracts/BasePoolConfig.sol:89]: html is generated. At some point, it'll be updated.

Comment (ljiatu) [reply contracts/BasePoolConfig.sol:89]: So I should remove all these?

Comment (PlayJok3r) [reply contracts/BasePoolConfig.sol:89]: No, I meant typos in HTML files would be fixed if we merge this pr and regenerate them. As @wisdant is the primary owner of the contracts, he will need to review and sign off this PR.

**Review by wisdant — CHANGES_REQUESTED** [review]
We will address these comments in our V2 contract. No need to make the head of the branch as unaudited for non-critical changes.

**Pull request closed without merging**